stream_alert/rule_processor/threat_intel.py

```diff
@@ -19,17 +19,16 @@
 import boto3
 from boto3.dynamodb.types import TypeDeserializer
 from botocore.exceptions import ClientError, ParamValidationError
-from netaddr import IPAddress
-
+from netaddr import IPAddress, IPNetwork
+from netaddr.core import AddrFormatError
 from stream_alert.shared import NORMALIZATION_KEY
 from stream_alert.shared.backoff_handlers import (
     backoff_handler,
     success_handler,
     giveup_handler
 )
-
 from stream_alert.rule_processor import LOGGER
-
+from helpers.base import in_network
```

Comment:
I know @chunyong-lin suggested this but I actually would not do this. These helpers were intended for use with `rules` and not core code.. If this is a requirement within threatintel, I'd suggest reimplementing a helper in this class. we can talk more since you've been pulled in multiple directions on this

Reply:
alternatively we could put some helper in `stream_alert/shared/..` that could be imported